### ISSUE TITLE
Blender 4.0 does not recognize ZLUDA

### ISSUE THREAD

**Issue by Kizaing** [vosen/ZLUDA#101]
Hi, hoping I can get some insight on this or maybe I'm doing something wrong haha

I am using OpenSUSE Tumbleweed with an RX 7800XT GPU, using the standard Mesa kernel driver.

I extracted the ZLUDA release zip and I added it as an environment variable to Blender like so:

`'LD_LIBRARY_PATH=/home/<user>/Documents/zluda/:$LD_LIBRARY_PATH'`

I have also tried running it from the command line with `LD_LIBRARY_PATH="/home/<user>/Documents/zluda/:$LD_LIBRARY_PATH" blender`

And under Blender's Render Device settings there are no options available

![image](https://github.com/vosen/ZLUDA/assets/1566109/61a962b9-36d8-4d35-b551-aecdbd890775)

Thanks!

**Comment by vosen** [comment]
Check here: https://github.com/vosen/ZLUDA/blob/master/TROUBLESHOOTING.md

**Comment by deuszex** [comment]
Using ```./zluda.exe -- <my_blender_path>/blender.exe``` gets me Render Device availablity, but even after switching to the right I will use my CPU. No errors anywhere, just uses CPU instead.
![image](https://github.com/vosen/ZLUDA/assets/30974986/23340fde-7449-4568-b991-9251c0275fc7)

**Comment by DGdev91** [comment]
> Using `./zluda.exe -- <my_blender_path>/blender.exe` gets me Render Device availablity, but even after switching to the right I will use my CPU. No errors anywhere, just uses CPU instead.

Have you selected "GPU Compute" in the rendering options for your scene?
![immagine](https://github.com/vosen/ZLUDA/assets/428479/33116acb-a38a-45b1-bbab-e14d90ca1139)

**Comment by deuszex** [comment]
> Have you selected "GPU Compute" in the rendering options for your scene?

Yeah I did. Even restarted the process and the machine itself too multiple times. Went as far as taking a long benchmark scene to see whether it uses CPU to render or it's just a ZLUDA compile. GPU never went above 4% but most CPU cores maxed out. Although GPU dedicated memory was occupied (around 10 gigs instead of the usual 3 gigs).

**Comment by vosen** [comment]
What happens if you render from the command line? I used this command (adjust for you blender and zluda paths)
```
c:\dev\zluda\target\debug\zluda_with.exe -- "C:\Program Files\Blender Foundation\Blender 3.4\blender.exe" -d --debug-cycles -noaudio --enable-autoexec -b "C:\dev\blender_benchmark\benchmark\bmw27\bmw27_gpu.blend" -x 1 -F PNG -f 1 -- --cycles-device CUDA
```

**Comment by deuszex** [comment]
Nope, nothing. But I might have overlooked something.
![image](https://github.com/vosen/ZLUDA/assets/30974986/2d7a8b74-f423-4fc0-95da-88702ec46c0f)
![image](https://github.com/vosen/ZLUDA/assets/30974986/ffc84b73-6c08-4beb-a8c5-f5dd88376465)


Although I might have something.
![image](https://github.com/vosen/ZLUDA/assets/30974986/9e2a8505-4c8a-4fbc-a512-449f7c1aabbb)

It might only get the compute cores which are not displayed as usage for whatever reason. Does the same from command line, and using HIP, same speed too.

**Comment by deuszex** [comment]
Tested the same scene render on 3080ti with native CUDA. ZLUDA 7900xtx finished faster. This seems to be a hardware resource monitoring issue.
